app/controllers/schools/setup_school_cohort_controller.rb

```diff
@@ -59,9 +59,15 @@ def what_changes
     end
 
     def what_changes_confirmation
+      what_changes_choice = @setup_school_cohort_form.what_changes_choice
```

Comment:
I think this should be moved inside the `send_fip_programme_changed_email!` method as it's not used in the controller method.
The `send_fip_programme_changed_email!` method should be called with no arguments then.

Reply:
I've pushed the suggested changes